File: ISP/Kata_ Shops Multirepository/SolucionPhp/Shop/MiniShop.php
<?php

declare(strict_types=1);

namespace SolucionPhp\Shop;

class MiniShop
{
    private string $name;
    private int $phone;

    public function __construct(string $name, int $phone)
    {
        $this->name = $name;
        $this->phone = $phone;
    }

    public function getName(): string
    {
        return $this->name;
    }

    public function getPhone(): int
    {
        return $this->phone;
    }
}

File: ISP/Kata_ Shops Multirepository/SolucionJava/Shop/MiniShop.java
package SolucionJava.Shop;

public class MiniShop {
    private String name;
    private Integer phone;

    public MiniShop(String name, Integer phone){
        this.name = name;
        this.phone = phone;
    }

    public String getName(){
        return this.name;
    }

    public Integer getPhone(){
        return this.phone;
    }
}
